draft-ietf-taps-interface.md

```diff
@@ -738,7 +738,12 @@ a Preconnection will only support calling Listen(), not Initiate(). The accepted
 are receive-only.
 
 Similarly, specifying a multicast group address on the Remote Endpoint will indicate that the
-resulting connection will be used to send multicast messages.
+resulting connection will be used to send multicast messages, and that the Preconnection will
+support Initiate() but not Listen(). The Connection created this way is send-only.
+
+No semantics for creating bidirectional connections for multicast group members or the use of
+Rendezvous() with Preconnections that have endpoints containing group addresses are defined in
+this document.
 
```

Comment:
Not sure I agree with this. We need to either prohibit the use of `Rendezvous()` with multicast in this version of the API, or define some semantics for what its use means.

Reply:
I'm OK just not to do this. I'd be against delaying this until we've thought it through - maybe there'll be more to add in future? so much of TAPS has actually been implemented, I'd not be in favour of adding last-minute things as we finish.

Reply:
Hmm, fair point.  Best if different implementations don't use it in different incompatible ways.

I guess one option is to replace this with something like "A Rendezvous() call on Preconnections containing group addresses results in an EstablishmentError as described in {{rendezvous}}.", and another option is to leave it implicit and remove this sentence.

Reply:
Hmm, fair point.  Best if different implementations don't use it in different incompatible ways.

I guess one option is to replace this with something like "A Rendezvous() call on Preconnections containing group addresses results in an EstablishmentError as described in {{rendezvous}}.", and another option is to leave it implicit and remove this sentence.

Reply:
It seems to me that "A Rendezvous() call on Preconnections containing group addresses results in an EstablishmentError as described in {{rendezvous}}." addresses what @csperkins says but removing the sentence doesn't. So, this suggested sentence seems to be the better option?